docs/plugins/albumtypes.rst

```diff
@@ -0,0 +1,57 @@
+AlbumTypes Plugin
+=================
+
+The ``albumtypes`` plugin adds the ability to format and output album types,
+such as "Album", "EP", "Single", etc. List of available type can be found
+`here`_.
+
+To use the ``albumtypes`` plugin, enable it in your configuration
+(see :ref:`using-plugins`). Then, add ``$atypes`` to your path formats as
+desired.
```

Comment:
"The plugin defines a new field `$atypes`, which you can use in your path formats or elsewhere."